packages/provider_devtools_extension/lib/src/instance_viewer/instance_viewer.dart

```diff
@@ -118,15 +120,18 @@ class _InstanceViewerState extends ConsumerState<InstanceViewer> {
 
   Iterable<Widget> _buildError(
     Object error,
-    StackTrace? _,
+    StackTrace? stack,
     InstancePath __,
   ) {
     if (error is SentinelException) {
       final valueAsString = error.sentinel.valueAsString;
       if (valueAsString != null) return [Text(valueAsString)];
     }
 
-    return const [Text('<unknown error>')];
+    extensionManager.postMessageToDevTools(DevToolsExtensionEvent(
+        DevToolsExtensionEventType.unknown,
+        data: {'stacktrace': stack.toString(), 'error': error.toString()}));
+    return [Text('<unknown error>\n$stack')];
```

Comment:
```suggestion

    return [Text('<unknown error>\n$stack')];
```

Reply:
I don't like the idea of showing the Stack like that. That's a multiline message on a single-line UI

And if we're going to show the stack, we should show the error.